### DISCUSSION TITLE
Lightroom Classic CC Works!

### DISCUSSION THREAD

**Discussion by ngxson** [LinSoftWin/Photoshop-CC2022-Linux#59] [Show and tell]
Thanks to your script, I'm now having Lightroom Classic CC running on the same setup (same `drive_c`) folder with Photoshop CC 2021.

![image](https://user-images.githubusercontent.com/7702203/183441185-875f8025-5d52-49aa-8aec-2336200dc0c1.png)

For anyone interested in doing the same thing:

1. Download this: https://1drv.ms/u/s!AqHhN3byznPZpSAA-nAIaD3Uxznk?e=eCWCFb  
Extract to `(your photoshop path)/drive_c/Program Files/Adobe`
2. Copy `(your photoshop path)/drive_c/launcher.sh` to `(your photoshop path)/drive_c/lightroom.sh`
3. Open `lightroom.sh` in a text editor:
4. Change .exe file location to `drive_c/Program\ Files/Adobe/Adobe\ Lightroom\ Classic\ CC/Lightroom.exe`
5. Optional: Open `~/.local/share/applications/` and create your own `lightroom.desktop` based on `photoshop.desktop`

**Comment by MiMillieuh** [comment]
Well that's cool,

Nice work ^^

**Comment by FoxStevenHY** [comment]
WOW!!!
Thanksssss!!!!!!!!!!!!!!
